.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules/*
-
+**node_modules**
+s
 **..DS_Store
```

Comment:
typo, extra full stop, sorry didn't notice this before, should be **.DS_Store 

It should then ignore them everywhere, and then remove the extras from line 12